src/NServiceBus.AcceptanceTests/Performance/TimeToBeReceived/When_TimeToBeReceived_used_with_unobtrusive_mode.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace NServiceBus.AcceptanceTests.Performance.TimeToBeReceived
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_TimeToBeReceived_used_with_unobtrusive_mode : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Message_should_not_be_received()
```

Comment:
not too happy with the naming, it misses a "when" to be a really true statement?